experimental/packages/opentelemetry-instrumentation-xml-http-request/src/xhr.ts

```diff
@@ -339,7 +339,7 @@ export class XMLHttpRequestInstrumentation extends InstrumentationBase<XMLHttpRe
       kind: api.SpanKind.CLIENT,
       attributes: {
         [SemanticAttributes.HTTP_METHOD]: method,
-        [SemanticAttributes.HTTP_URL]: url,
+        [SemanticAttributes.HTTP_URL]: parseUrl(url).toString(),
```

Comment:
we have several call sites of `parseUrl` in the instrumentation. I believe we can parse the URL at the beginning of the span creation, and keep using that parsed URL to avoid duplicated parsing.

Reply:
The usages of pareUrl() looks like they are processing a past in property -- So I don't think they should be changed.